changelog/993.breaking.rst

```diff
@@ -0,0 +1 @@
+The output if an exception occurs during the plugin import in conftest is now similar to the output if the exception is directly in conftest. All exceptions during conftest's startup are now ExitCode.INTERNAL_ERROR. Previously it was ExitCode.USAGE_ERROR.
```

Comment:
```suggestion
Exceptions raised while importing `conftest.py` files now cause pytest to exit with `ExitCode.INTERNAL_ERROR` instead of `ExitCode.USAGE_ERROR`.

Also, exceptions that happen during plugin import in `conftest.py` files are now similar to the output obtained from exceptions raised directly in `conftest.py` files. 
```

Reply:
I have already restored ``ExitCode.USAGE_ERROR`` after @asottile comment. Should it ultimately be ``ExitCode.INTERNAL_ERROR``?